### DISCUSSION TITLE
How would I go about animating the exit state of the hover card?

### DISCUSSION THREAD

**Discussion by rajbirjohar** [radix-ui/primitives#2202] [Help] [answered]
Pretty much the title.

I tried using [data-state]="closed" but this is applied on the trigger it seems. I tried using framer-motion as well but was running into issues due to it being active on hover instead of on click or something like that to toggle the state. Any pointers?

**Comment by joaom00** [comment, marked as answer]
Example with CSS https://codesandbox.io/p/sandbox/upbeat-hodgkin-5ll4fq?file=%2Fstyles.css%3A19%2C1-24%2C2

**Comment by snelsi** [comment]
[Delegating unmounting for JavaScript Animation](https://www.radix-ui.com/docs/primitives/guides/animation#delegating-unmounting-for-javascript-animation)

[Framer Motion AnimatePresence](https://www.framer.com/motion/animate-presence/)

**Comment by rajbirjohar** [comment]
Thanks! This solved it.
